src/DynamoCoreWpf/ViewModels/Menu/PreferencesViewModel.cs

```diff
@@ -900,7 +900,11 @@ public string ExpanderActive
                 {
                     expanderActive = value;
                     OnPropertyChanged(nameof(ExpanderActive));
-                }          
+                }
+                else
+                {
+                    expanderActive = "";
```

Comment:
String.empty

Reply:
@QilongTang 
Fixed in the next commit: https://github.com/DynamoDS/Dynamo/pull/11717/commits/aebc3ae78e4a04c76849d4044fc8d48ba2e8df84